lnrpc/rpc.proto

```diff
@@ -1936,6 +1936,9 @@ message PolicyUpdateRequest {
 
     /// The required timelock delta for HTLCs forwarded over the channel.
     uint32 time_lock_delta = 5 [json_name = "time_lock_delta"];
+
+    /// The maximum HTLC size in milli-satoshis.
+    uint64 max_htlc_msat = 6 [json_name = "max_htlc_msat"];
```

Comment:
I don't think this field is meant to be changed after channel creation? 🤔

Reply:
talked to @Roasbeef and seems to think it should still be included. I submitted this feature as a separate PR: #2438.